playingaway.R

```diff
@@ -8,134 +8,157 @@ library(ggplot2)
 library(ggthemes)
 library(tidyr)
 
+# Get team data ----
+team_state_csv <-
+    "team,state
```

Comment:
Sorry, you are right. I added the states. I prefer using full names.

I was writing this code to test two hypotheses:
- Victoria teams don't travel as often as they should: Given 8/18 teams are
non-Victorian and Victorian teams collectively sell one-teams worth of home
games to interstate venues, each Victorian team should play 6 of their away
games interstate. Interstate teams should play 6 of their away games in
Victoria. It turns out that they travel about the right amount.
- Winning away against top eight teams is rare. It is. Only 2/32 teams
playing away against top eight teams have won this year.



On 18 May 2016 at 10:22, Ian Gow <notifications@github.com> wrote:

> In playingaway.R
> <https://github.com/zerogetsamgow/afltables/pull/1#discussion_r63627461>:
>
> > @@ -8,134 +8,157 @@ library(ggplot2)
> >  library(ggthemes)
> >  library(tidyr)
> >
> > +# Get team data ----
> > +team_state_csv <-
> > +    "team,state
>
> Possibly. Rename state here to state_abbrev and then merge with this:
>
> states <- read_csv("state_abbrev,state
>     NSW,New South Wales
>     TAS,Tasmania,
>     WA,Western Australia
>     NT,Northern Territory
>     SA,South Australia
>     QLD,Queensland
>     ACT,Australian Capital Territory
>     VIC,Other (New Zealand, etc.)")
>
> I'd be surprised if you need to use state from the underlying source very
> often: teams and venues have states, but what else? (Games are played by
> teams at venues, so games don't have states directly.)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/zerogetsamgow/afltables/pull/1/files/1e1cf49a9c0163ce81bd805d451e45cdc8f81014#r63627461>
>